src/Features/LightLimitFix.cpp

```diff
@@ -711,24 +696,24 @@ void LightLimitFix::UpdateLights()
 					light.radius /= (float)clusteredLights;
 					light.positionWS[eyeIndex] /= (float)clusteredLights;
 
-					currentLightCount += AddCachedParticleLights(lightsData, light, 2.0f, eyeIndex);
+					currentLightCount += AddCachedParticleLights(lightsData, light, particleLight.second.second, 1.0f, eyeIndex);
```

Comment:
Just wanted to note you've changed the dimmer multiple. These multiples were from your original code.

Reply:
I changed this due to it being incorrect.

I can make a separate commit after the merge to completely remove this argument.